components/bootloader_support/include/esp_flash_partitions.h

```diff
@@ -38,7 +40,9 @@ extern "C" {
 /* The md5sum value is found this many bytes after the ESP_PARTITION_MAGIC_MD5 offset */
 #define ESP_PARTITION_MD5_OFFSET 16
 
+#ifndef CONFIG_MCUBOOT_ESPRESSIF
```

Comment:
can we replace this with more generic define, for example `#ifndef __ZEPHYR__`  ?

Reply:
When building Espressif Port from Zephyr build system, `__ZEPHYR__` is already defined by the environment, so it is not possible.